dev/tasks/tasks.yml

```diff
@@ -1788,6 +1791,12 @@ tasks:
         UBUNTU: 18.04
       run: ubuntu-docs
 
+  ############################## vcpkg tests ##################################
+
+  test-build-vcpkg-win:
```

Comment:
Should we run this job nightly?
We can run this job nightly by adding this to `nightly:` group.

Reply:
For now it might be better to run it only when we do release candidate verification.

A next step will be to get an Arrow 3.0.0 "port" (that's the term they use) added to vcpkg, so users can simply `vcpkg install arrow` in their projects that use the Arrow C++ library as a dependency. Currently there is an [old port of Arrow on vcpkg](https://github.com/microsoft/vcpkg/tree/master/ports/arrow) and to date we have not treated vcpkg like a formal release channel. Once that is done, we will want to run nightly tests that install Arrow with vcpkg onto one or more supported platforms.

Reply:
OK.
But I don't want to increase tasks that are ran only when release candidate verification. Normally, these tasks are outdated and they increase release cost. I hope that we can go to the next step by the next release.